src/main/webapp/js/instructorFeedbacks.js

```diff
@@ -214,6 +214,53 @@ function bindCopyEvents() {
     });
 }
 
+function setupFsCopyModal() {
+    $('#fsCopyModal').on('show.bs.modal', function(event) {
+        var button = $(event.relatedTarget); // Button that triggered the modal
+        var actionlink = button.data('actionlink');
```

Comment:
This function seems to be duplicated several times throughout now (in instructorFeedbacks.js, instructorHome.js and instructorFeedbackEdit.js) that it might make sense for it to go into instructor.js.